scripts/is_coveralls_configured.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+/*
+Check if coveralls configuration is available.
+
+Exit 0 if configured to run coveralls
+Exit 1 if coveralls will fail
+
+For required environment variables, see:
+https://github.com/nickmerwin/node-coveralls
+
+Coveralls for Ruby also supports .coveralls.yml
+*/
+
+const { existsSync } = require('node:fs')
+
+if (process.env.COVERALLS_REPO_TOKEN) {
+  console.log('coveralls configured with environment variable COVERALLS_REPO_TOKEN.')
+  process.exit(0)
+} else if (existsSync('.coveralls.yml')) {
+  console.log('coveralls configured with configuration file ".coveralls.yml".')
```

Comment:
Do we need a `process.exit(0)` for the second condition?  Or just put both conditions in the first block?

Reply:
The default is an exit code of 0, I'm going to just be explicit on the failure path and drop `process.exit(0)`